proto/src/main/proto/org/dependencytrack/notification/v1/notification.proto

```diff
@@ -216,6 +218,11 @@ message ProjectVulnAnalysisCompleteSubject {
   string token = 4;
 }
 
+message UserPrincipalSubject {
+  string username = 1;
+  string email = 2;
```

Comment:
`email` is labeled as `optional` in the Proto definition of the API server PR. It should be consistent.